app/controllers/jobs_controller.rb

```diff
@@ -41,8 +29,8 @@ def index
     end
 
     # Get the complete lists of live and dead jobs from the server
-    raw_live_jobs = getCurrentJobs
-    raw_dead_jobs = getDeadJobs
+    raw_live_jobs = TangoClient.tango_jobs()["jobs"]
```

Comment:
If you always read ["jobs"] from the result of tango_jobs, should we just return that value instead?